### DISCUSSION TITLE
This page can't be fetched throwing an error

### DISCUSSION THREAD

**Discussion by gerroon** [wallabag/wallabag#7058] [Q&A]
Hi

I am using self hosted version and trying to fetch this in Brave. it throws an error. I normally do not have an issue with this kind of stuff. Does anyone know what the issue is?

https://iep.utm.edu/maurice-blondel/

**Comment by HolgerAusB** [comment]
I have no problem with this article. Self-hosted and app.wallabag.it are loading both the full article, So if that persist, we need some logs and a more detailed report, how you tried that.

**Reply by gerroon** [reply]
Thanks for checking, I am using the Chrome Wallabagger. Let me see if I can get some logs.
